### DISCUSSION TITLE
There is no mention of case-sensitivity anywhere in this repo... or maybe I cannot find it?

### DISCUSSION THREAD

**Discussion by jessekrubin** [oramasearch/orama#584] [General]
Search (AFAICT) is case sensitive. How can I turn off case-sensitivity?

**Comment by micheleriva** [comment]
Case-sensitive search is not currently supported

**Reply by jessekrubin** [reply]
might be good to mention in docs? others might find it helpful to know. 

When i grep the repo I don't see any mention at all of case-sensitivity.

**Reply by allevo** [reply]
Would you be interested in sending a PR to mention this in the documentation?

**Reply by jessekrubin** [reply]
@micheleriva @allevo boom: https://github.com/oramasearch/orama/pull/585

**Reply by jessekrubin** [reply]
btw your repo is not clone-able on windows
